### PR TITLE
Inverse fft node

### PR DESCRIPTION
**Pull request by Orisu179**
### Description

- [x] I've included a concise description of what each node does

### Styleguide

- [x] My node adheres to the [styleguide](https://github.com/flojoy-io/nodes/blob/main/node_styleguide.md) for Flojoy nodes

### Docs

- [x] I've submitted a PR for a documentation page for the new node(s) that contains usage examples (see docs.flojoy.io)

### CODE REVIEW

File: TRANSFORMERS/SIGNAL_PROCESSING/FFT/FFT.py [hunk 1]
```diff
@@ -39,26 +40,42 @@ def FFT(dc_inputs: list[DataContainer], params: dict) -> DataContainer:
...
+    else:  # used for processing
+        fourier = fft.rfft(signal_value) if real else fft.fft(signal_value)
+        frequency = x
 
-    result = abs(fourier)
-    return DataContainer(x=frequency, y=result)
+    return (
+        DataContainer(x=frequency, y=fourier)
+        if display
+        else DataContainer(x=frequency, y={"real": fourier.real, "imag": fourier.imag})
```
Comment (smahmed776) [inline TRANSFORMERS/SIGNAL_PROCESSING/FFT/FFT.py:80]: Do we currently have support for dict type in 'y' key?

Comment (Orisu179) [reply TRANSFORMERS/SIGNAL_PROCESSING/FFT/FFT.py:80]: I don't think so, but I couldn't think of a better way to pass complex numbers to the next node. If you know any other alternatives please lmk

File: GENERATORS/SIMULATIONS/SINE/SINE.py [hunk 2]
```diff
@@ -20,7 +20,7 @@ def SINE(dc_inputs: list[DataContainer], params: dict) -> DataContainer:
         print("invalid waveform passed as param, using default:", waveform)
 
     if waveform == "sine":
-        y = Y0 + A * np.sin(np.radians(2 * np.pi * F) * x + np.radians(PHASE))
+        y = Y0 + A * np.sin(2 * np.pi * F * x + np.radians(PHASE))
```
Comment (scorinaldi) [inline GENERATORS/SIMULATIONS/SINE/SINE.py:23]: Could you break this out into a separate PR? This doesn't seem related to the FFT nodes.

**Review by scorinaldi — CHANGES_REQUESTED** [review]
Thanks for getting these nodes made! I only have a few minor notes. 

Also, could you please add unit tests for `FFT.py` and `IFFT.py`? You can see an example of such a unit test here: https://github.com/flojoy-io/nodes/blob/c28808f2d0aec8f116cdb3fccb46c4f5256574e9/TRANSFORMERS/ARITHMETIC/ADD/ADD_test_.py

**Review by scorinaldi — APPROVED** [review]
LGTM
